ynr/apps/people/api/next/filters.py

```diff
@@ -0,0 +1,16 @@
+from django_filters import filterset, filters
+
+from people.models import Person
+
+
+class PeopleFilterMixin(filterset.FilterSet):
+    class Meta:
+        model = Person
+        fields = ["last_updated"]
+
+    last_updated = filters.DateTimeFilter(
```

Comment:
Was having a play around with this locally, and see what you meant about the date formatting being a bit awkward. [Then I came across the](https://django-filter.readthedocs.io/en/stable/ref/filters.html#isodatetimefilter) `IsoDateTimeFilter` which I think we should use instead of the `DateTimeFilter` - as then when we use the API in WCIVF, it will mean we can use the `isoformat()` helper method on a datetime, rather than having to format the date ourselves.

Reply:
Should I do this for all filters expecting an ISO datetime?

Reply:
Thinks its good on just the new ones for now, as dont want to break anything by changing the pre-existing ones.